pconvert_rust/test/blending.js

```diff
@@ -0,0 +1,57 @@
+const fs = require("fs");
+const path = require("path");
+const assert = require("assert");
+const pconvert = require("pconvert-rust");
+
+const TEST_ASSETS = path.resolve("assets/test");
+
+describe("NodeJS WASM", async function() {
+    it("should have known module constants", () => {
+        const constants = pconvert.getModuleConstants();
+        const keys = [
+            "ALGORITHMS",
+            "COMPILATION_DATE",
+            "COMPILATION_TIME",
+            "COMPILER",
+            "COMPILER_VERSION",
+            "COMPRESSION_TYPES",
+            "FEATURES",
+            "FILTER_TYPES",
+            "LIBPNG_VERSION",
+            "PLATFORM_CPU_BITS",
+            "VERSION"
+        ];
+        keys.forEach(k => assert(k in constants));

```

Comment:
`k in constants` is not doing what you think is doing :D. One of the weird Javascript decisions is that arrays are objects where the keys are the indexes.

Go with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
In any case, wouldn't `deepStrictEquals` do what you want with these 2 lines?

Reply:
Yup!